Source/Manager/PasswordlessTokenStore.swift

```diff
@@ -70,37 +61,13 @@ struct PasswordlessTokenStore {
         }
 
         let components = data.components(separatedBy: ":")
-
-        guard components.count == 2 && !components[0].isEmpty && !components[1].isEmpty else {
+        guard let token = components.first else {
             throw PasswordlessTokenStoreError.invalidData(connection, data)
```

Comment:
I think this could happen if the user upgrades the app in the middle of a passwordless login, at which point there'll be a failure but when the user tries to login again, the old data will be re-written. Or I've missed something else?

Reply:
Cool, I missed the fact that it would be overwritten before attempting to obtain it👌